src/lib/sw_engine/tvgSwCommon.h

```diff
@@ -321,7 +321,7 @@ bool strokeParseOutline(SwStroke* stroke, const SwOutline& outline);
 SwOutline* strokeExportOutline(SwStroke* stroke, SwMpool* mpool, unsigned tid);
 void strokeFree(SwStroke* stroke);
 
-bool imagePrepare(SwImage* image, const Matrix* transform, const SwBBox& clipRegion, SwBBox& renderRegion, SwMpool* mpool, unsigned tid);
+bool imagePrepare(SwImage* image, Polygon* triangles, uint32_t count, const Matrix* transform, const SwBBox& clipRegion, SwBBox& renderRegion, SwMpool* mpool, unsigned tid);
```

Comment:
suggest more clear name:
i.e) count -> trianglesCnt
Other parameter names in functions as well.

Reply:
@hermet I agree - I actually just used count because I saw it used in other functions, like `Result colorStops(const ColorStop* colorStops, uint32_t cnt) noexcept;` but triangleCount is better.